opm/autodiff/BlackoilModelParameters.cpp

```diff
@@ -64,9 +65,11 @@ namespace Opm
     void BlackoilModelParameters::reset()
     {
         // default values for the solver parameters
-        dp_max_rel_      = 1.0e9;
+        dp_max_rel_      = 1.0;
```

Comment:
I'm guessing that this change to the default can be partially responsible for the increase in Newton iteration counts, do you think that is true?